contracts/erc20-gateway/ERC20Cogateway.sol

```diff
@@ -35,14 +35,12 @@ contract ERC20Cogateway is
     /**
      * @notice It initializes ERC20Cogateway contract.
      *
-     * \pre Gateway is not activated.
-     *
      * \post Calls `MessageOutbox.setupMessageOutbox` and
      *       `MessageInbox.setupMessageInbox` with genesis* values read
      *       from `GenesisERC20Cogateway` contract.
      */
     function setup()
-        external
+        public
```

Comment:
This function should be moved under 
/* Public Functions */

Reply:
Done.

Reply:
It is still under `/* External Functions */`